docs/user/gke-sre.md

```diff
@@ -206,6 +230,53 @@ A verbose cheatsheet can also be found [here](https://kubernetes.io/docs/referen
 
   - `$ kubectl top nodes`
 
+## GKE Load Testing
+
+If you'd like to perform some performance testing, troubleshooting GKE related issues,
+or would like to test out a new features capability within GKE, a load test script is
+available for use within `k8s/tools/load-test.sh`. Prior to running, please ensure you
+review the script and update any variables for your test. Most importantly, the load test
+script does not currently support the creation of test GCP disks and would need to be created
+prior to running the script. By default, the script will look for GCP disks with the naming
+convention of `<DISK_NAME-i>`, `i` being a range of `1` and `MAX_DISKS`. Once test data has
+been created, you can run the script on any machine or pod that has the Turbinia client
+installed and configured to the correct Turbinia GKE instance. Please run the following
+command to execute the load test.
+
+```
+./k8s/tools/load-test.sh /PATH/TO/LOADTESTS
```

Comment:
What is the the /path/to/loadtests here?  Is that an output directory I guess?  Maybe clarify what that should be?

Reply:
Ack. Updated to make it a bit more clear to what the path is